src/components/TaskBoard/index.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from "react";
+import { useSelector } from "react-redux";
+import AddTaskCard from "components/TaskCard/AddTaskCard.jsx";
+import style from "components/TaskBoard/index.module.scss";
+import TaskList from "components/TaskList";
+
+const TaskBoard = () => {
+  const [isCardCreated, setIsCardCreated] = useState(false);
```

Comment:
As this state will only be responsible for showing and hiding todo creation card, we can name this more appropriately

```suggestion
  const [showCreateCard, setShowCreateCard] = useState(false);
```